rules/windows/image_load/image_load_susp_dbghelp_dbgcore_load.yml

```diff
@@ -52,7 +52,7 @@ detection:
       - '-k UnistackSvcGroup -s WpnUserService'
   filter2:   # Not available in Sysmon, but in Aurora
     CommandLine:
-      - 'C:\WINDOWS\winsxs\*\TiWorker.exe -Embedding'
+      - 'C:\WINDOWS\winsxs\\*\TiWorker.exe -Embedding'
       - 'C:\Windows\system32\svchost.exe -k netsvcs -p -s wuauserv'
       - 'C:\Windows\System32\svchost.exe -k WerSvcGroup'
   condition: (signedprocess or unsignedprocess) and not filter1 and not filter2
```

Comment:
why not : 
```yaml
  filter2:   # Not available in Sysmon, but in Aurora
    CommandLine:
      - 'C:\Windows\system32\svchost.exe -k netsvcs -p -s wuauserv'
      - 'C:\Windows\System32\svchost.exe -k WerSvcGroup'
  filter3:
    CommandLine|startswith: 'C:\WINDOWS\winsxs\'
    CommandLine|endswith: '\TiWorker.exe -Embedding'
  condition: (signedprocess or unsignedprocess) and not 1 of filter*
```

Reply:
I just fixed the "issue" but you are right. This is the better solution.